api/pkg/app/app.go

```
@@ -275,25 +288,16 @@ func initDB() (*Database, error) {
 }
 
 // initLogger returns a instance of SugaredLogger depending on the EnvMode
-func initLogger(mode EnvMode) (*zap.SugaredLogger, error) {
-
-	var log *zap.Logger
-	var err error
+func initLogger(mode EnvMode) *log.Logger {
 
 	switch mode {
 	case Production:
-		log, err = zap.NewProduction()
-
+		l, _ := zap.NewProduction()
```

should we ignore the error here?